example/Set-height-width-for-chatwidget.html

```
@@ -0,0 +1,48 @@
+			}
+		};
+		var s = document.createElement("script");
+		s.type = "text/javascript";
+		s.async = true;
+		s.src = "https://widget.kommunicate.io/v2/kommunicate.app";
+		var h = document.getElementsByTagName("head")[0];
+		h.appendChild(s);
+		window.kommunicate = m;
+		m._globals = kommunicateSettings;
+	})(document, window.kommunicate || {}); 
+</script>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </script> ]

(tag-pair)